subprojects/core/src/integTest/groovy/org/gradle/api/tasks/TaskInputPropertiesIntegrationTest.groovy

```diff
@@ -559,21 +559,12 @@ task someTask(type: SomeTask) {
         """
         expect:
         executer.expectDeprecationWarning().withFullDeprecationStackTraceDisabled()
-        if (expectToFail) {
-            // Singular null outputs will cause build to fail, but message should still be printed
-            fails "test"
-        } else {
-            succeeds "test"
-        }
+        succeeds "test"
```

Comment:
This looks like a breaking change. Are we okay with that?

Reply:
I would say yes, since we break less builds.

Reply:
OK, it makes sense to do it like this.